Test/comp/firstSteps/Calls-F.dfy

```diff
@@ -0,0 +1,12 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:py "%s" >> "%t"
```

Comment:
As a side note, since this is a simple test for a compiler, would you be able to make all the other compilers pass this test as well? e.g. Duplicate this line and add js, java, cpp, cs and go.

Reply:
so for context, This is a test file is made up from already present test file Calls.dfy where this file is only testing for 1 function from it because that is what is implemented in the python compiler, I was thinking adding all the languages here will just add repetitive tests.

Reply:
So the plan on the long run is that you will remove this file and add python to the tests of `//calls.dfy` when it will be able to run it?
If so, please add a comment to this file indicating the purpose, such as:
```suggestion
// Once the python compiler will be able to run the file call.dfy, we can remove this file
// RUN: %dafny /noVerify /compile:4 /compileTarget:py "%s" >> "%t"
```

Reply:
I think the plan is to keep these files for more compilers development. I will add a comment to it.

Reply:
Yup, the intention is for `firstSteps` to contain an ordered sequence of very simple test files for every new compiler to handle. It will provide a more incremental breakdown of features than the current set of compilation test cases. In the next PR, we should also move `IKnowThisMuchIs.dfy` into this folder and label it as the very first file to attempt. We'll probably want to name the files with prefixes like `0-IKnowThisMuchIs.dfy` to make that even clearer.